console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -68,12 +68,14 @@ class UserController {
 
   // search each choosen span elements and set title manually
   // to display roles description on hover
-  setTitle () {
-    if (this.systemRoles) {
-      let selectEls = $('li.search-choice')
-      $(selectEls).each(i => {
-        let text = selectEls[i].firstChild.innerHTML
-        selectEls[i].firstChild.setAttribute('title', this.systemRoles[text])
+  setTitles () {
+    if (this.roleDescriptions) {
+      let selectEls = document.querySelectorAll('li.search-choice span')
+      selectEls.forEach(span => {
```

Comment:
Now that you don't use jQuery selectEls is a nodeList. `forEach` on nodeList is [not implement in IE](https://developer.mozilla.org/fr/docs/Web/API/NodeList/forEach#Compatibilit%C3%A9_des_navigateurs).

You can use  `[].forEach.call` to be IE-proof.
```javascript
[].forEach.call(
  document.querySelectorAll('li.search-choice span'),
  span => span.setAttribute('title', this.roleDescriptions[text.innerHTML])
)
```

Reply:
> orEach on nodeList is not implement in IE

+1 for jQuery ;)